assets/css/style.css

```diff
@@ -386,3 +386,11 @@ h5[data-field="name"]:before,
 .navbar-toggler-icon {
     background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(0, 0, 0, 0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E");
 }
+
```

Comment:
Below code should be removed. You can use bootstrap classes to set text transformation e.g.:  `font-italic`, `font-weight-bold` etc.

Reply:
You can select relevant lines by shift-clicking the line numbers @ciastektk 

![image](https://user-images.githubusercontent.com/3183926/111971751-f6af3000-8afc-11eb-855c-9da497975dcb.png)

Reply:
Thanks @Steveb-p! It will be very useful 😄